### PR TITLE
Fix for cmaes

### PR DESCRIPTION
**Pull request by costashatz**
Minor fix in cmaes wrapper.. We were resetting to fixed values the initial point..

### CODE REVIEW

File: src/limbo/tools/random_generator.hpp [hunk 1]
```diff
@@ -122,7 +122,7 @@ namespace limbo {
         /// - we use a C++11 random number generator
         Eigen::VectorXd random_vector_unbounded(int size)
         {
-            static rgen_gauss_t rgen(0.0, 100.0);
```
Comment (jbmouret) [inline src/limbo/tools/random_generator.hpp]: Does "unbounded" means between 0 and 10 in Limbo random numbers?

**Comment by jbmouret** [comment]
Nope, but the documentation should at least say it explicitly.

**Comment by costashatz** [comment]
> Nope, but the documentation should at least say it explicitly.

OK. I will add something..

**Comment by costashatz** [comment]
Should I merge?

**Comment by jbmouret** [comment]
yes.

**Comment by costashatz** [comment]
**Merging...**

**Pull request merged**